### ISSUE TITLE
Initiating audio voice or video voip calls fails without proper error notification but receiving calls works

### ISSUE THREAD

**Issue by jittygitty** [vector-im/element-web#22378]
### Steps to reproduce

1. Where are you starting? What can you see?
I click to initiate outgoing audio voice/voip call or video.
Nothing happens, no error nothing happens in the UI.

Underneath, the ERROR is:
```
Access to XMLHttpRequest at 'https://srv.redacted.com/_matrix/client/r0/thirdparty/user/im.vector.protocol.sip_virtual?native_mxid=%40USER2%3Asrv.redacted.com' from origin 'vector://vector' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: It does not have HTTP ok status.
index.js:218   GET https://srv.redacted.com/_matrix/client/r0/thirdparty/user/im.vector.protocol.sip_virtual?native_mxid=%40USER2%3Asrv.redacted.com net::ERR_FAILED
```
The ABOVE error is seen in LOGS "each time" that I click the phone or video icon in the Element Desktop App.

Which makes SENSE because loading the following URL gives **404 page not found**:
https://srv.redacted.com/_matrix/client/r0/thirdparty/user/im.vector.protocol.sip_virtual?native_mxid=%40USER2%3Asrv.redacted.com
"404 page not found"

Probably it would be good to "catch" such errors and throw a **hint in the GUI and not fail silently**.

Otherwise PLEASE tell your users what I had to find out on my own:

That Desktop Element App is an Electron app and can be debugged by doing Ctrl-Shift-I which is how I clearly saw the above errors, each and every time I clicked on the phone icon or the video icon, while the UI did "nothing" and showed no error whatsoever, and text messaging worked perfectly fine both ways etc.

PREVIOUSLY, I believe this was working, I can't say 100% because I'm not sure if I only initiated non-DM conference room phone/video before, but I "think" I had initiated such Direct-Message calls before from Desktop App so I'm not sure if an update to Desktop app or Dendrite server broke it. 

Also, NOTEWORTHY: Element Android App logged into same Dendrite homeserver, "can" initiate phone call to this same Desktop Element user just fine. So it seems that the "404 page not found", should not really be a deal breaker in having audio/video work in DM p2p direct-message rooms if it works initiating from Android.

Maybe most don't see the error since they either use unaffected mobile apps, OR they use a homeserver running SYNAPSE, which apparently doesn't give a 404 page not found, for example:
https://matrix.org/_matrix/client/r0/thirdparty/user/im.vector.protocol.sip_virtual?
gives:
`{"errcode":"M_MISSING_TOKEN","error":"Missing access token"}`

Unfortunately, in my previous interaction here with project maintainers, even after I asked a couple of times how to enable dev mode on Desktop App, I was never told how to do so or if it was possible etc. 

If they had told me I could simply do a Ctrl-Shift-I and then see what errors popup when I click the phone/video icons, then we would have avoided a lot of noise and unnecessary unpleasantness in the original tickets on this issue which got closed, and so I could not update them.

(Part of "other" issues I raised affecting Android app problem of initiating calls was addressed by Dendrite in PR https://github.com/matrix-org/dendrite/pull/2485  clarifying this will help follow issues below better etc.)

ORIGINAL tickets for this issue affecting Element DESKTOP:

https://github.com/vector-im/element-web/issues/22331
https://github.com/matrix-org/dendrite/issues/2495

#### What did you expect?
When clicking Audio or Video icon in Element Desktop app, to have something happen. Ideally for the call to initiate and start ringing.
#### What happened instead?
Nothing. Absolutely nothing. No error, no popup message. 
Above, in Steps to Reproduce, I provided the actual relevant error on phone/video button press.

### Operating system

Windows 10

### Application version

Element version: 1.10.13 Olm version: 3.2.8

### How did you install the app?

https://element.io/get-started

### Homeserver

Dendrite version":"0.8.5+a53c9300

### Will you send logs?

Yes

**Comment by jittygitty** [comment]
I have just alerted Dendrite team of the above, see:
https://github.com/matrix-org/dendrite/issues/2499

**Comment by spaetz** [comment]
Duplicate of https://github.com/vector-im/element-web/issues/21680 I guess?

**Comment by t3chguy** [comment]
Yup

Duplicate of #21680
